build-aux/snap/snapcraft.yaml

```diff
@@ -411,7 +419,24 @@ parts:
       - snapd
     override-pull: |
       craftctl default
-      if [ -f fips-build ]; then
+      # FIPS stamp files were created in the 'snap' part build
+      if [ -f "${CRAFT_PROJECT_DIR}/fips-build-lp" ]; then
+        mkdir -p etc/apt/preferences.d/
+        # need to up the priority of the repository
+        # TODO what if we need to choose between FIPS and FIPS preview?
+        cat <<-'EOF' > /etc/apt/preferences.d/fips.pref
+      Package: *
+      Pin: release o=LP-PPA-fips-cc-stig-fips-under-certification
+      Pin-Priority: 1010
+      EOF
```

Comment:
Spacing here looks odd. Are you sure it is okay? I would have expected the lines to be aligned the same as CAT (especially that you use `<<-`).

After checking the man page I suspect those are tabs below. Can you please add a comment to let people know that tabs need to be preserved in editing?